salt/fileclient.py

```diff
@@ -465,7 +465,7 @@ def get_url(self, url, dest, makedirs=False, saltenv='base',
         url_path = os.path.join(
                 url_data.netloc, url_data.path).rstrip(os.sep)
 
-        if url_scheme.lower() in 'abcdefghijklmnopqrstuvwxyz':
+        if url_scheme.lower() in 'abcdefghijklmnopqrstuvwxyz' and url_scheme != '':
```

Comment:
An empty string should evaluate to false, so maybe just `if url_scheme and url_scheme.lower() in 'abcdefghijklmnopqrstuvwxyz':`? Same for `parsed_scheme` if that works?